<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 57, 57);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>&lt;x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>600</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>600</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>600</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>730</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>730</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(197, 197, 197);
font-family: arial;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 57, 57);
text-align: right;
color: rgb(197, 197, 197);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:40pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:20pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
